Automattic/jetpack
# Tiled Galleries: avoid the use of extract

#### Changes proposed in this Pull Request:
﻿
Let's avoid using `extract()` in our tiled gallery templating.

#### Jetpack product discussion

* p3btAN-1xB-p2
* D67289-code

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Dashboard > Modules
* Ensure the Tiled Galleries feature is active.
* Create a new post.
* Add a classic block.
* Insert a gallery into that block.
* Pick a tiled gallery type (square, tiled, ...)
* See that they're displayed properly on the frontend when you publish your post.
